src/m_misc.c

```diff
@@ -2319,6 +2322,20 @@ default_t defaults[] = {
 #endif
 
 #if defined(_WIN32)
+  {
```

Comment:
Disclaimer: I have no idea what you guys are doing here.

But, do we really need another three config variables? I mean, I'd consider myself a rather "advanced user", but still I have absolutely no clue what these are all about. How should the avarage user know what's the right choice here and what consequence a bad choice would have? Is there anything like a reasonable default that applies everywhere?

Reply:
This is for MIDI enthusiasts - you need to buy a vintage sound card (Roland SC-55 or something) and make it work on a modern system, I don't know how.
What I've done:
1. Install the Windows MIDI Mapper selection tool (VirtualMIDISynth)
2. Install the loopMIDI driver.
3. Set up a simple VST host or use a DAW.
4. Get some VST MIDI synthesizer plugin. Roland has a [Sound Canvas VA](https://www.roland.com/us/products/rc_sound_canvas_va/) product, I recommend the [Yamaha S-YXG50](https://veg.by/en/projects/syxg50/) ("abandonware").

Once you install and configure all of this, these settings will start to make sense 😄 Maybe we should even add a SysEx option to the menu. We develop software for enthusiasts and we have them in our DW thread.

Reply:
Okay, this is clearly your (plural) ground and I rely on your expertise. I'm out.
![giphy](https://user-images.githubusercontent.com/6449237/205255582-a5cae501-b3eb-41c2-9e44-c885de3f646f.gif)